src/main/java/com/ericsson/ei/controller/RuleTestControllerImpl.java

```diff
@@ -85,8 +86,13 @@
             final HttpServletRequest httpRequest) {
         if (testEnabled) {
             try {
-                String aggregatedObject = ruleTestService.prepareAggregatedObject(
-                        new JSONArray(body.getListRulesJson()), new JSONArray(body.getListEventsJson()));
+                String aggregatedObject = StringUtil.EMPTY_STRING;
+                try {
```

Comment:
why this was written inside another try?

Reply:
It is require, as in sub try/catch we have to catch Exception and we are not returning anything. If we use one try/catch then we need to add finally block to return ResponseEntity, and Finally block is considered to be cleanup block, return is not generally expected in it. Hence, nested try/catch is require here.